README.md

```
@@ -68,350 +68,349 @@ docker compose -f docker-compose.sqlite.yml up -d
 
 ```bash
 cd frontend && npm ci && npm run dev
 ```
 
-Vite 会自动代理 `/api` 和 `/health` 到后端，开发时访问 `http://localhost:5173/admin/`。
+Vite will automatically proxy `/api` and `/health` to the backend. During development, visit `http://localhost:5173/admin/`.
 
 ---
 
-## 环境配置
+## Environment Setup
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)